.travis.yml

```diff
@@ -174,7 +174,6 @@ cache:
     ccache: true
     directories:
         - boost_1_57_0
-        - build
```

Comment:
Ah you mean it does not include the build directory but it does include the ccache directory?